core/src/codegen/from_meta_impl.rs

```diff
@@ -89,22 +113,22 @@ impl ToTokens for FromMetaImpl<'_> {
                     }
                 };
 
-                let word_or_err = variants
+                let from_word_method = variants
                     .iter()
                     .find_map(|variant| {
```

Comment:
Not necessary right now, but might be worth exploring generating the `from_word` method in `FromMetaOptions` to simplify this code generation.

Reply:
I went ahead and did this; it does make the `codegen` piece simpler to read, which I like.